localstripe/resources.py

```diff
@@ -1099,7 +1103,7 @@ def __init__(self, customer=None, subscription=None, metadata=None,
         except AssertionError:
             raise UserError(400, 'Bad request')
 
-        Customer._api_retrieve(customer)  # to return 404 if not existant
+        cus = Customer._api_retrieve(customer)  # to return 404 if not existant
```

Comment:
Look at other `# to return 404 if not existant` comments: they're here only when the call isn't assigned to any variable :wink:

Reply:
[Not always](https://github.com/adrienverge/localstripe/blob/6e62d74465d7fea87fed28b42caf245fa95711d0/localstripe/resources.py#L3093) but it sounds weird now indeed, I hesitated. I will remove it.